### ISSUE TITLE
deleted pages still show up

### ISSUE THREAD

**Issue by skshetry** [iterative/dvc.org#1405]
Eg: [`metrics` subcommands that were removed recently still appear](https://dvc.org/doc/command-reference/metrics/add
).

It shows 404 at first, and then fetches from cache somewhere and displays the old data.

**Comment by jorgeorpinel** [comment]
Weird! Looks like a deployment cache issue because the document is gone gone from the code base.

**Comment by utkarshsingh99** [comment]
https://github.com/gatsbyjs/gatsby/issues/2410

https://github.com/gatsbyjs/gatsby/issues/10844

Gatsby has closed and reopened this issue many times. Maybe running `gatsby clean` in the deployment server could solve this problem. 
Or, another solution written in these issues is to manually clear the `.cache` folder. (Although I don't see how these solutions are way different)

I surfed through the public folder in my local clone and here's what I found in `public/page-data/doc/command-reference/metrics`:
![Screenshot from 2020-06-17 12-28-53](https://user-images.githubusercontent.com/32031518/84865814-769ff980-b096-11ea-92aa-ea55be578004.png)

Maybe it isn't deleted unless we clear the cache.
